src/components/RunHeader/RunHeader.js

```diff
@@ -44,16 +44,23 @@ class RunHeader extends Component {
           }
           return (
             runName && (
-              <h1>
-                <div className="block-icon">
-                  {getStatusIcon({ reason, status })}
-                </div>
-                <div className="run-name" title={runName}>
-                  {runName}
-                </div>
-                <span className="status-label">{reason}</span>
-                <span className="time">{lastTransitionTime}</span>
-              </h1>
+              <div>
+                <h1>
+                  <div className="block-icon">
+                    {getStatusIcon({ reason, status })}
+                  </div>
+                  <div className="run-name" title={runName}>
+                    {runName}
+                  </div>
+                  <span className="status-label">{reason}</span>
+                  <span className="time">{lastTransitionTime}</span>
+                  <Rebuild
+                    {...this.props}
```

Comment:
Does the Rebuild button require all of the props from RunHeader or just a subset?

Reply:
I'm passing 

```
    setShowRebuildNotification={this.setShowRebuildNotification}
          setShowRebuildNotificationKind={this.setShowRebuildNotificationKind}
          setShowRebuildNotificationLogsURL={this.setShowRebuildNotificationLogsURL}
          setShowRebuildNotificationMessage={this.setShowRebuildNotificationMessage}
```

to the RunHeader from PipelineRun which Rebuild needs to access, so unfortunately yeah 😢